common/util/rviz_plugins/autoware_vehicle_rviz_plugin/src/tools/jsk_overlay_utils.cpp

```diff
@@ -127,11 +125,15 @@ void OverlayObject::updateTextureSize(unsigned int width, unsigned int height)
 {
   const std::string texture_name = name_ + "Texture";
   if (width == 0) {
-    RCLCPP_WARN(logger_, "width=0 is specified as texture size");
+    RCLCPP_WARN(
+      rclcpp::get_logger("OverlayObject"),
+      "width=0 is specified as texture size");
     width = 1;
   }
   if (height == 0) {
-    RCLCPP_WARN(logger_, "height=0 is specified as texture size");
+    RCLCPP_WARN(
+      rclcpp::get_logger("OverlayObject"),
+      "height=0 is specified as texture size");
```

Comment:
[Question] Why `logger` dependency was removed?

Reply:
On second thoughts, this is more simple than passing logger to class.

Reply:
@wep21 So do you mean that
- in https://github.com/tier4/AutowareArchitectureProposal.iv/pull/246 you fixed to inject `logger_`
- but now you change your mind it was not necessary
Is this right?